drivers/flash/flash_stm32_qspi.c

```diff
@@ -221,10 +343,46 @@ static int qspi_read_sfdp(const struct device *dev, off_t addr, uint8_t *data,
 		.AddressMode = QSPI_ADDRESS_1_LINE,
 		.DataMode = QSPI_DATA_1_LINE,
 	};
+#else
```

Comment:
Here the delta is too big, so I understand you left this alone.

Reply:
IMO we should change either all commands or nothing.